test-requirements.txt

```diff
@@ -3,3 +3,4 @@ pytest==7.2.0
 pytest-cov==4.0.0
 flake8==6.0.0
 responses==0.22.0
+grpcio==1.53.0
```

Comment:
why not fix it in requirements itself. Kind of missing dependency.

Reply:
@pritamghanghas That's on purpose. I've removed this dependency so I can use the most convenient one in the target container. For hm-gatewayrs, it's 1.50.1, hm-diag, 1.53.0 so on so forth. This way we can avoid unnecessary grpcio builds.